src/components/v2v3/shared/SplitItem/SplitAmountValue.tsx

```diff
@@ -60,7 +60,9 @@ export function SplitAmountValue({ props }: { props: SplitProps }) {
         <span>
           (
           <CurrencySymbol currency={curr} />
-          {splitValueFormatted}
+          {props.showFees
+            ? splitValueFormatted
+            : formatWad(splitValue, { precision: 0 })}
```

Comment:
**Question:** Why `precision: 0`? Discrete tokens are still possible, and we should account for possibly showing an ETH value in this component when `showFees` is false.

Reply:
You have a good point, however aren't we only in this situation when `showFees` is false accounts for these values for reserved tokens? If so, ETH value would never display up. Am I missing something? 

With the line `const reservedTokensFormatted = formatWad(reservedTokens, "precision: 0")` , we are precisioning values in the `DistributeReservedTokensModal` to 0.